one/src/main.rs

```diff
@@ -210,6 +255,15 @@ impl Daemon {
         let srv_handle = srv.handle();
         tokio::spawn(srv);
 
+        let api_bind_address = self.api_bind_address.clone();
+        // Start Ceramic API
+        let network = self.network.clone();
+        tokio::spawn(ceramic_api::start(
```

Comment:
Does this mean we need a thread safe mutex on recon?

Reply:
Yes that is why we are using `Arc<Mutex<Recon>>`